k8s/prod/moki.yml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: moki-deployment
+  labels:
+    app: moki
+  namespace: tekst-prod
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: moki
+  template:
+    metadata:
+      labels:
+        app: moki
+    spec:
+      containers:
+        - name: moki
+          image: <image>
+          ports:
+            - name: app-port
+              containerPort: 4200
+          imagePullPolicy: Always
+          livenessProbe:
+            httpGet:
+              path: /moki
+              port: 80
+            initialDelaySeconds: 30
+            periodSeconds: 20
+            failureThreshold: 3
+            timeoutSeconds: 5
+          readinessProbe:
+            httpGet:
+              path: /moki
+              port: 80
+            initialDelaySeconds: 30
+            periodSeconds: 15
+            failureThreshold: 3
+            timeoutSeconds: 1
+
+---
+
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: moki-ingress
+  namespace: tekst-prod
+spec:
+  ingressClassName: nginx
+  rules:
+    - host: <host_url>
+      http:
+        paths:
+          - path: /moki
+            pathType: Prefix
+            backend:
+              service:
+                name: moki-service
+                port:
+                  number: 80
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: moki-service
+spec:
+  ports:
+    - port: 4200
```

Comment:
Husker ikke hvordan det var med portene men på stage er det motsatt, altså
```    
    - port: 80
      targetPort: 4200
```
skulle det vært det her og?

Reply:
Denne er samme som stage for moki, tenker du på stage for en annen app?

Reply:
Heh, var visst slik i en gammel commit... Jeg trodde jeg hadde pullet men tydligvis hadde den feilet i pull uten at jeg fikk det med deg.. Da er all good!